test/extension.test.ts

```diff
@@ -1,10 +1,14 @@
-import * as assert from "assert";
+import chai from "chai";
```

Comment:
[C] This is the file to focus on. The reason that I'd recommend using the `expect` syntax is that is the closest that we can get to emulating `jest` (most popular Js testing framework) whist using `mocha`.